### PR TITLE
Feature: Phenomodels language adjustment (#2337)

### PR DESCRIPTION
**Pull request by JessicaHillert**
This PR adds a functionality or fixes a bug.
OR
This PR marks a new Scout release. We apply semantic versioning. This is a major/minor/patch release for reasons.

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
There is a test (tests/server/blueprints/institutes/test_institute_views.py) that fails at [this line](https://github.com/Clinical-Genomics/scout/blob/7edeb27fc210b8309c993a02c794d4abbb33a67b/tests/server/blueprints/institutes/test_institute_views.py#L207). That's because the subpanel title is no longer named "Subpanel title". You'll need to modify this test to have the tests passing :)

File: scout/server/blueprints/institutes/forms.py [hunk 1]
```diff
@@ -132,12 +132,12 @@ class PhenoModelForm(FlaskForm):
 
     model_name = TextField("Phenotype panel name", validators=[validators.InputRequired()])
     model_desc = TextField("Description", validators=[validators.Optional()])
-    create_model = SubmitField("create")
+    create_model = SubmitField("Create")
```
Comment (northwestwitch) [inline scout/server/blueprints/institutes/forms.py:135]: 👍

File: scout/server/blueprints/institutes/forms.py [hunk 2]
```diff
@@ -132,12 +132,12 @@ class PhenoModelForm(FlaskForm):
...
 
 class PhenoSubPanelForm(FlaskForm):
     """A form corresponfing to a phenopanel sub-panel"""
 
-    title = TextField("Subpanel title", validators=[validators.InputRequired()])
-    subtitle = TextField("Subpanel subtitle", validators=[validators.Optional()])
-    add_subpanel = SubmitField("save subpanel")
+    title = TextField("Phenotype subpanel title", validators=[validators.InputRequired()])
+    subtitle = TextField("Phenotype subpanel subtitle", validators=[validators.Optional()])
+    add_subpanel = SubmitField("Save phenotype subpanel")
```
Comment (northwestwitch) [inline scout/server/blueprints/institutes/forms.py:143]: 👍

**Review by northwestwitch — CHANGES_REQUESTED** [review]
Good job! Much better now! Fine to merge as soon as the tests pass.

File: CHANGELOG.md [hunk 3]
```diff
@@ -23,6 +23,12 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Refactor according to CodeFactor - mostly reuse of duplicated code
 
 
+## [4.31.2]
```
[the remark this answers is not in the document]
Comment (JessicaHillert) [reply CHANGELOG.md]: Fixed!

**Review by northwestwitch — APPROVED** [review]
Perfect, thanks!

**Review by northwestwitch — APPROVED** [review]

**Pull request merged**